scripts/seed-database/write/flows.sql

```diff
@@ -46,7 +46,7 @@ SET
   version = EXCLUDED.version,
   settings = EXCLUDED.settings,
   copied_from = EXCLUDED.copied_from,
-  analytics_link = null;
+  analytics_link = NULL;
```

Comment:
Sorry for bundling, but this is a followup syntax fix to #2414

Reply:
Confirmed this works here https://github.com/theopensystemslab/planx-new/actions/runs/6852179815